spark-cassandra-connector/src/test/scala/com/datastax/spark/connector/cql/CassandraConnectorConfSpec.scala

```diff
@@ -0,0 +1,14 @@
+package com.datastax.spark.connector.cql
+
+import org.apache.commons.lang3.SerializationUtils
+import org.apache.spark.SparkConf
+import org.scalatest.{Matchers, FlatSpec}
+
+class CassandraConnectorConfSpec extends FlatSpec with Matchers {
+
+  it should "be serializable" in {
+    val conf = CassandraConnectorConf.apply(new SparkConf(loadDefaults = true))
```

Comment:
CassandraConnectorConf.apply ? Why not just 

         CassandraConnectorConf(new SparkConf())

Because also SparkConf loadDefaults true is the default: def this() = this(true)